tests/libtest/test_ibmcloud.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf8 -*-
+
+
+import logging
+
+from ocs_ci.ocs.platform_nodes import IBMCloud
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import libtest
+from ocs_ci.ocs import constants, node
+from ocs_ci.ocs.node import get_all_nodes
+from ocs_ci.framework.pytest_customization.marks import ibmcloud_platform_required
+
+from ocs_ci.utility.utils import run_cmd
+
+logger = logging.getLogger(__name__)
+
+
+@libtest
+@ibmcloud_platform_required
+def test_rebootnodes():
+    """
+    Check basic consistency in platform handling.
+    """
+    ibmcloud = IBMCloud()
+    worker_nodes = node.get_nodes(node_type="worker", num_of_nodes=1)
+    ibmcloud.restart_nodes(worker_nodes)
+
+@libtest
+@ibmcloud_platform_required
+def test_attachvolume():
+    """
+    Check basic consistency in platform handling.
+    """
+    ibmcloud = IBMCloud()
+
+    volume = ibmcloud.getVolumeId()
+
+    worker_nodes = node.get_nodes(node_type="worker", num_of_nodes=1)
+    ibmcloud.attach_volume(volume, worker_nodes)
```

Comment:
This test should actually use the fixture to detach/delete this volume attached during this test. You can take a look here: https://stackoverflow.com/questions/44716237/pytest-passing-data-for-cleanup